src/annotator/index.js

```diff
@@ -66,11 +66,16 @@ function init() {
 
   config.pluginClasses = pluginClasses;
 
-  const annotator = new Klass(document.body, config);
+  const guest = new Guest(document.body, config);
```

Comment:
Would it be better to instantiate `guest` inside `sidebar`?

Reply:
There may be cases in future where the guest is specialized or configured depending on the document type and it will probably make sense to do that outside the sidebar. There may also be scenarios where the Sidebar is instantiated without a guest, if the content to be annotated is inside a frame.